hooks/src/index.js

```diff
@@ -26,15 +26,45 @@ let oldBeforeUnmount = options.unmount;
 const RAF_TIMEOUT = 100;
 let prevRaf;
 
+let mask = 0;
+let localIdCounter = 0;
+
 options._diff = vnode => {
```

Comment:
We can make this assumption because RTS calls this https://github.com/preactjs/preact-render-to-string/blob/master/src/index.js#L137 and follows a consistent diffing compared to our core algorithm